core/server/middleware/validation/blog-icon.js

```diff
@@ -61,7 +61,7 @@ module.exports = function blogIcon() {
 
         // CASE: file should not be larger than 100kb
         if (!validIconSize(req.file.size)) {
-            return next(new errors.RequestEntityTooLargeError({message: i18n.t('errors.api.icons.fileSizeTooLarge', {extensions: iconExtensions})}));
+            return next(new errors.ValidationError({message: i18n.t('errors.api.icons.invalidFile', {extensions: iconExtensions})}));
```

Comment:
Yes. A `RequestEntityTooLargeError` will cause a different error message in the client. And as we want to have one unified error message, I had to change the type.